projects/adrv9001/Makefile

```diff
@@ -1,12 +1,13 @@
 TINYIIOD ?= n
+
+include ../../tools/scripts/generic_variables.mk
+
```

Comment:
As CFLAGS are initialized in generic variable makefile we can't use before it. One thing we can do is we can add project specific FLAGS in src.mk file. I have initialized CFLAGS in  generic variable makefile to avoid repetition of CFLAGS.

Reply:
Ok, let's move initialization of those flags then to src.mk, I'm ok with that